index.html

```diff
@@ -0,0 +1,553 @@
+              ><img src="images/twitter.png" alt="twitter icon"
+            /></a>
+          </li>
+          <li class="head__icons__li">
+            <a
+              href="https://www.linkedin.com/in/itumeleng-rantsane-552a44244/"
+              class="head__icons__link link"
+              ><img src="images/Linkedin.svg" alt="linkedin icon"
+            /></a>
+          </li>
+          <li class="head__icons__li">
+            <a href="#" class="head__icons__link link"
+              ><img src="images/M.png" alt="M icon"
+            /></a>
+          </li>
+          <li class="head__icons__li">
+            <a
+              href="https://github.com/TumiRantsane"
+              class="head__icons__link link"
+              ><img src="images/github.svg" alt="github icon"
+            /></a>
+          </li>
+          <li class="head__icons__li">
+            <a href="#" class="head__icons__link link"
+              ><img src="images/hand.svg" alt="hand icon"
+            /></a>
+          </li>
+        </ul>
+      </section>
+
+      <section class="works" id="works">
+        <ul class="works__cards ul grid">
+          <li class="works__card flex bg-white">
+            <img
+              class="works__card__img"
+              src="images/tonic-project-desktop.png"
+              alt="tonic project"
+            />
+            <div class="works__card__text">
+              <h2 class="works__card__title fw-700 fz-32 lh-44 clr-n-800">
+                Tonic
+              </h2>
+              <div class="works__card__subtitle flex">
+                <p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">
+                  CANOPY
+                </p>
+                <ul class="works__card__subtitle-ul flex">
+                  <li
+                    class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500"
+                  >
+                    Back End Dev
+                  </li>
+                  <li
+                    class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500"
+                  >
+                    2015
+                  </li>
+                </ul>
+              </div>
+              <p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="works__card__topics flex">
+                <p
+                  class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50"
+                >
+                  html
+                </p>
+                <p
+                  class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50"
+                >
+                  css
+                </p>
+                <p
+                  class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50"
+                >
+                  javaScript
+                </p>
```

Comment:
- [ ] As requested by the previous reviewer, it is semantically correct to put this as a list. Ordered or un-ordered. This is an example of how you are required to make it

```
<ul>
  >li class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">HTML</li>
  ....
</ul>
```
**Note:** Kindly implement this on all project works cards as me repeating the same message would be redundant. The next reviewer will check whether you implemented this on all the cards as I have requested.